application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Macro.xml

```diff
@@ -665,7 +660,33 @@ if (typeof(XWiki) == "undefined" || typeof(XWiki.widgets) == "undefined" || type
   }) // end PlainList extends View
 
   jQuery.fullCalendar.views.plainList = PlainList;
-  
+  /**
+  * Events triggered before deleteEvents modal is shown: save the button that triggers
+  * the modal, close MoccaCalendarPopup, update the name of event on displayed text.
+  */
+  jQuery(document).on('show.bs.modal', '#deleteEvents', function(event) {
+    var relatedTarget = $(event.relatedTarget);
+    jQuery(this).data('relatedTarget', $(relatedTarget));
+    var calendarPopup = jQuery(relatedTarget).data('calendarPopup');
+    calendarPopup.closeDialog();
+    var eventName = '«' + calendarPopup.interactionParameters.event.title.escapeHTML() + '»';
```

Comment:
I believe ``escapeHTML()`` is from Prototype.js so you're mixing jQuery with Prototype.js. The problem with this is that we'll have issues when removing Prototype.js because we'll have to review all the jQuery code to see if it uses Prototype.js APIs.

Moreover, if you're calling ``text()`` below then I don't think you need to escape the HTML.